Sources/Datadog/RUM/Instrumentation/Views/RUMViewsHandler.swift

```diff
@@ -35,14 +26,18 @@ internal final class SwiftUIRUMViewsHandler: SwiftUIViewHandler {
     /// The current date provider.
     private let dateProvider: DateProvider
 
+    /// `UIKit` view predicate. `nil`, if `UIKit` auto-instrumentations is
+    /// disabled.
+    internal var predicate: UIKitRUMViewsPredicate?
+
     /// The notification center where this handler observe the following notifications:
     /// - `UIApplicationDidEnterBackgroundNotification`
     /// - `UIApplicationWillEnterForegroundNotification`
     private weak var notificationCenter: NotificationCenter?
 
     /// The RUM Command subscriber responsible for processing
     /// this publisher's commands.
-    private weak var subscriber: RUMCommandSubscriber?
+    internal weak var subscriber: RUMCommandSubscriber?
```

Comment:
Does it need to be `internal`? Now `RUMViewsHandler` state can be altered from outside.

Reply:
I think this was for testing, let me double check!

Reply:
So the SwiftUI view handler wasn't exposing the `subscriber`, but the [`UIKitRUMViewsHandler` did](https://github.com/DataDog/dd-sdk-ios/blob/master/Sources/Datadog/RUM/AutoInstrumentation/Views/UIKitRUMViewsHandler.swift#L38). This is required to be able to run [this test](https://github.com/DataDog/dd-sdk-ios/blob/master/Tests/DatadogTests/Datadog/RUM/AutoInstrumentation/RUMAutoInstrumentationTests.swift#L23L45).

I agree it would be best to keep it private but this would prevent from checking it in tests. WDYT?

Reply:
Ah I see 👍. I think it's all good until:
* it's `private` and has integration tests coverage;
* it's `internal` and has unit tests coverage.

We have both unit and integration coverage, so all options are fine. We can keep it `internal` 👍.